tsconfig.json

```diff
@@ -25,6 +25,7 @@
     ]
   },
   "include": [
-    "./src/**/*.ts"
+    "./src/**/*.ts",
+    "node_modules/@types/puppeteer/index.d.ts"
```

Comment:
You probably don't want to include this explicitly; @types should automatically be scanned for definitions.